lib/OpenQA/WebAPI/Controller/API/V1/Table.pm

```diff
@@ -320,4 +284,47 @@ sub destroy {
     $self->render(json => {result => int($ret)});
 }
 
+=over 4
+
+=item prepare_settings()
+
+Internal method to prepare settings when add or update admin table.
+Use by both B<create()> and B<update()> method.
```

Comment:
If this is an internal method then the name should be `_prepare_method`. Or is there another reason it's part of the public API?

Reply:
yes, this is an internal method. I will change the method name